modules/packages/LinearAlgebra.chpl

```diff
@@ -2063,6 +2319,176 @@ proc kron(A: [?ADom] ?eltType, B: [?BDom] eltType) {
   return C;
 }
 
+/*
+  Matrix exponential using Pade approximation. This method returns N*N matrix which
+  is Matrix exponential of `A`
+
+  :arg A: Expects an N*N square matrix.
+  :type A: 2D Array
+
+  :arg useExactOneNorm: boolean value specifying if the onenorm has to be exact. Defaults to true.
+  :type useExactOneNorm: bool
+
+.. note::
+
+  [1] Awad H. Al-Mohy and Nicholas J. Higham (2009)
+  "A New Scaling and Squaring Algorithm for the Matrix Exponential."
+  SIAM Journal on Matrix Analysis and Applications.
+  31 (3). pp. 970-989. ISSN 1095-7162
+
+  :throws LinearAlgebraError: If Input Matrix is not Square Matrix.
+
+  :returns: Matrix exponential of the given matrix.
+  :rtype: `A`
```

Comment:
One last nit!  You want to move it so that the `.. note::` is the very last thing in this documentation comment.  Right now it's putting the `throws` and `returns` tags in the note itself and we don't want that.  Otherwise, this looks perfect, thanks!  And it passes testing